src/Contextually/Relevant.cs

```diff
@@ -61,6 +67,96 @@ public static IDisposable Info(NameValueCollection info, string containerName =
                 return container.Info(info);
             }
         }
+
+        /// <summary>
+        /// Starts a new Info block with a set of values.
+        /// </summary>
+        /// <param name="name">The name of a value.</param>
+        /// <param name="value">The value.</param>
+        /// <returns>An <see cref="IDisposable"/> object for use with a using block.</returns>
+        public static IDisposable Info(string name, string value) =>
+            Info(new NameValueCollection(1)
+            {
+                [name] = value
+            });
+
+        /// <summary>
+        /// Starts a new Info block with a set of values.
+        /// </summary>
+        /// <param name="tuple">The name-value tuple this Info block represents.</param>
+        /// <returns>An <see cref="IDisposable"/> object for use with a using block.</returns>
+        public static IDisposable Info((string name, string value) tuple) =>
+            Info(new NameValueCollection(1)
+            {
+                [tuple.name] = tuple.value
+            });
+
+        /// <summary>
+        /// Starts a new Info block with a set of values.
+        /// </summary>
+        /// <param name="tuple1">A name-value tuple this Info block represents.</param>
+        /// <param name="tuple2">A name-value tuple this Info block represents.</param>
+        /// <returns>An <see cref="IDisposable"/> object for use with a using block.</returns>
+        public static IDisposable Info(
+            (string name, string value) tuple1,
+            (string name, string value) tuple2) =>
+            Info(new NameValueCollection(2)
+            {
```

Comment:
Why bother with this versus params?

Reply:
The memory allocation. `params` is a syntactic sugar that transpiles into an array.